choreolib/src/main/java/choreo/auto/AutoFactory.java

```diff
@@ -34,7 +34,7 @@
  */
 public class AutoFactory {
   static final AutoRoutine VOID_ROUTINE =
-      new AutoRoutine(null, "VOID-ROUTINE", new EventLoop()) {
+      new AutoRoutine(null, "VOID-ROUTINE", new EventLoop(), () -> true) {
```

Comment:
This is unsafe, as `trajectoryCmd()` will never fail even if the alliance is unknown (will default downstream to run a blue side auto). Not sure if there's an easy approach here as VOID_ROUTINE could no longer be static, see my comment in `AutoTrajectory` about a potential alternative.

Additionally, `VOID-ROUTINE` should be private. It isn't used elsewhere in the `choreo.auto` package, or for unit tests.